tasks/utils.py

```diff
@@ -143,13 +143,14 @@ def get_payload_version():
                 continue
             pkgname = whitespace_split[0]
             if pkgname == "github.com/DataDog/agent-payload":
-                comment_split = line.split("//")
+                # Example of line: github.com/DataDog/agent-payload v4.40.0+incompatible
+                comment_split = re.split(r'[ +]', line)
```

Comment:
I find the name `comment_split` confusing since it's no longer refering to a comment, maybe `version_split`?